tests/job_applications/tests.py

```diff
@@ -776,38 +776,37 @@ def test_accept_trigger_manual_approval_without_hiring_end_at(self):
         email = job_application.email_manual_approval_delivery_required_notification(accepted_by)
         assert "Date de fin du contrat : Non renseigné" in email.body
 
-    def test_refuse(self):
-        # When sent by authorized prescriber.
+    @pytest.mark.parametrize("is_sent_by_proxy", [True, False])
+    @pytest.mark.parametrize("is_shared_with_job_weeker", [True, False])
```

Comment:
```suggestion
    @pytest.mark.parametrize("is_shared_with_job_seeker", [True, False])
```

(et pareil dans les arguments du test)